connectors-home/modules/ROOT/partials/debug-logs.adoc

```diff
@@ -0,0 +1,10 @@
+// Used in Connector troubleshooting pages.
+
+== Enable Verbose Exception Logging
+
+Enable verbose exception logs to show a complete stack trace instead of the default truncated output. 	
+
+To enable verbose exception logs:
```

Comment:
We can probably just delete line 7 and then end line 5 with : 
And then follow with the steps.